libpius/constants.py

```diff
@@ -1,11 +1,12 @@
 # vim:shiftwidth=2:tabstop=2:expandtab:textwidth=80:softtabstop=2:ai:
 
 import os
+import path
```

Comment:
did you mean .path?

Reply:
path is the name of a new file in the libpius folder

Reply:
Fixed it to be libpius.path instead of path.